src/components/header/renderer.tsx

```diff
@@ -188,6 +199,7 @@ class Header extends React.PureComponent<Props, State> {
                     <div
                       key={j}
                       onClick={() => {
+                        document.getElementById('run-button').click();
```

Comment:
This is not acceptable. We shouldn't be clicking buttons just to get things working. Look at the actions for loading a spec and the renderer.